src/Metadata/Driver/NullDriver.php

```diff
@@ -5,11 +5,23 @@
 namespace JMS\Serializer\Metadata\Driver;
 
 use JMS\Serializer\Metadata\ClassMetadata;
+use JMS\Serializer\Metadata\PropertyMetadata;
+use JMS\Serializer\Naming\PropertyNamingStrategyInterface;
 use Metadata\ClassMetadata as BaseClassMetadata;
 use Metadata\Driver\DriverInterface;
 
 class NullDriver implements DriverInterface
 {
+    /**
+     * @var PropertyNamingStrategyInterface
+     */
+    private $namingStrategy;
+
+    public function __construct(PropertyNamingStrategyInterface $namingStrategy)
```

Comment:
is this a bc break? (new required param)

Reply:
you are using this because now the `AnnotationOrAttributeDriver` can return null (becase of the  `$configured` thing...)

is it worth? what is AnnotationOrAttributeDriver returns the empty metadata as before?